2-mandatory/3-stocks.js

```diff
@@ -6,6 +6,23 @@
         - an array of arrays containing the closing price for each stock in each of the last 5 days.
             For example, CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS[2] contains the prices for the last 5 days for STOCKS[2] (which is amzn)
 */
+function getStock(stockTickers, closingPricesForAllStocks) {
+  let averages = [];
+
+  for (let i = 0; i < stockTickers.length; i++) {
+    let ticker = stockTickers[i];
+    let prices = closingPricesForAllStocks[i];
+    let sum = prices.reduce((total, price) => total + price);
```

Comment:
that's excellent - advanced coding here using 'reduce' and also arrow functions - very good!